deezerapp/src/app/app.component.html

```diff
@@ -1 +1,6 @@
-<h1>Works</h1>
+<app-header></app-header>
+<!-- <app-artist-search></app-artist-search> -->
```

Comment:
Please remove all comments from your source code, unless if they are conveying necessary information that is not apparent from the source code.